proprietary-specs.md

```diff
@@ -5,4 +5,11 @@ ensure that they are up to date with the current version of CloudEvents. That is
 the responsibility of the respective project maintainers.
 
 - [Apache RocketMQ Transport Binding](https://github.com/apache/rocketmq-externals/blob/master/rocketmq-cloudevents-binding/rocketmq-transport-binding.md)
-- [Google Cloud Pub/Sub Protocol Binding](https://github.com/google/knative-gcp/blob/master/docs/spec/pubsub-protocol-binding.md)
\ No newline at end of file
+- [Google Cloud Pub/Sub Protocol Binding](https://github.com/google/knative-gcp/blob/master/docs/spec/pubsub-protocol-binding.md)
+
+Want to add a binding to a proprietary transport?
+
+- Create a specification that follows the structure of the existing binding specifications (e.g. [http binding](http-protocol-binding.md)) - this will help SDK development.
```

Comment:
nit: Consider adding "or kafka binding".
Although they share the spec, it is easier to follow the Kafka's spec while implementing a protocol for an event driven system, since the HTTP one has a few implicit things (like the `Content-Type` header)

Reply:
fair point .. should I reference something like AMQP or NATS to avoid the whole 'partitioning' discussion ?